src/js/config/Config.test.js

```diff
@@ -4,7 +4,7 @@ var ConfigDev = {
   rootUrl: '',
   historyServer: '',
   analyticsKey: '39uhSEOoRHMw6cMR6st9tYXDbAL3JSaP',
-  stateRefresh: 2000,
+  stateRefresh: 20000,
```

Comment:
There is some sense in keeping it the same as in the application, but I also see the benefit of it.